src/lib/components/support.svelte

```diff
@@ -44,7 +44,18 @@
                 </p>
             {/if}
         </div>
-        {#if $organization?.billingPlan === BillingPlan.STARTER}
+        {#if $organization?.billingPlan === BillingPlan.PRO || $organization?.billingPlan === BillingPlan.SCALE}
```

Comment:
So, this is shown now if no org exists:

![image](https://github.com/appwrite/console/assets/1477010/1632a443-746c-4726-b342-50f0c99585b6)

But the "Get Premium support" button doesn't do anything.

Reply:
Also, what if I have a pro org, but I'm on the account page??